src/Application/Actions/Hooks/Stripe.php

```diff
@@ -15,7 +15,9 @@
 
 abstract class Stripe extends Action
 {
-    protected Event $event;
+    protected ?Event $event = null;
```

Comment:
As I created a new child of this class Psalm complained that Event was not set in constructor. So my solution was to initialise it to null here and insert null checks everywhere it's used.

Probably it would be better to remove the property entirely and replace it with local variables, but adding null checks seemed like an easier change.